harness/determined/experimental/client.py

```diff
@@ -209,17 +209,17 @@ def create_model(
 
 
 @_require_singleton
-def get_model(model_id: int) -> Model:
+def get_model(name: str) -> Model:
     """
     Get the :class:`~determined.experimental.client.Model` from the model registry
-    with the provided numeric id. If no model with that id is found in the registry,
+    with the provided name. If no model with that name is found in the registry,
     an exception is raised.
 
     Arguments:
-        model_id (int): The unique id of the model.
+        name (str): The unique name of the model.
     """
     assert _determined is not None
-    return _determined.get_model(model_id)
+    return _determined.get_model(name)
 
 
 @_require_singleton
```

Comment:
Can you remove this `get_model_by_name` too? (same reason, to avoid aliases)

(if I am correct that this has not been released yet...)

Reply:
`get_model_by_name` is now removed